### ISSUE TITLE
feature request: Add support for GraphicsMagick coalesce

### ISSUE THREAD

**Issue by Philipp-MOL** [avalanche123/Imagine#516]
While https://github.com/avalanche123/Imagine/blob/68d416ee2be3423c5758bddd31be11fcd80d16e5/lib/Imagine/Gmagick/Layers.php#L74 says that coalesce is not supported, the official GraphicsMagick documentation tells the opposite: http://www.graphicsmagick.org/GraphicsMagick.html#details-coalesce
We would love to see support for this function!

Best regards,
Philipp

**Comment by fritzmg** [comment]
The issue is that [PHP's `Gmagick`](https://www.php.net/manual/en/book.gmagick.php) (still) has no support for this.

**Comment by ausi** [comment]
> The issue is that [PHP's `Gmagick`](https://www.php.net/manual/en/book.gmagick.php) (still) has no support for this.

Are you sure? From the source code it looks like this is supported for quite some time: https://github.com/vitoc/gmagick/commit/7fc9e322902472801ba18708aedd428a192552d4

**Comment by fritzmg** [comment]
Indeed, then it seems the documentation on php.net is just out of date.
